Create a new directory throws an `internal server error`

I sent the following payload: 
```json
[
  {
    "action":"mediafile.create_directory",
    "data":[
      {
        "meeting_id":4,
        "title":"Rezepte für morgen",
        "parent_id":null
      }
    ]
  }
]
```

The backend throws the following internal server error:
```
backend_1           | DEBUG:openslides_backend.shared.handlers.base_handler:Perform action mediafile.create_directory.
backend_1           | DEBUG:openslides_backend.services.datastore.adapter:Start GET request to datastore with the following data: {"fqid": "user/1", "mapped_fields": ["organization_management_level", "group_$4_ids", "guest_meeting_ids"], "get_deleted_models": 1}
backend_1           | DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): datastore-reader:9010
datastore-reader_1  | 2021-06-14T08:56:58.454: [/app/reader/flask_frontend/json_handler.py] [INFO] GET-request: {'fqid': 'user/1', 'mapped_fields': ['organization_management_level', 'group_$4_ids', 'guest_meeting_ids'], 'get_deleted_models': 1}
datastore-reader_1  | 172.18.0.10 - - [14/Jun/2021 08:56:58] "POST /internal/datastore/reader/get HTTP/1.1" 200 -
backend_1           | DEBUG:urllib3.connectionpool:http://datastore-reader:9010 "POST /internal/datastore/reader/get HTTP/1.1" 200 66
backend_1           | DEBUG:openslides_backend.services.datastore.adapter:Get response with status code 200: {'organization_management_level': 'superadmin', 'group_$4_ids': [18]}
backend_1           | DEBUG:openslides_backend.services.datastore.adapter:Start RESERVE_IDS request to datastore with the following data: Collection: mediafile, Amount: 1
backend_1           | DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): datastore-writer:9011
datastore-writer_1  | 2021-06-14T08:56:58.470: [/app/writer/core/writer_service.py] [INFO] 1 ids reserved
datastore-writer_1  | 172.18.0.10 - - [14/Jun/2021 08:56:58] "POST /internal/datastore/writer/reserve_ids HTTP/1.1" 200 -
backend_1           | DEBUG:urllib3.connectionpool:http://datastore-writer:9011 "POST /internal/datastore/writer/reserve_ids HTTP/1.1" 200 12
backend_1           | DEBUG:openslides_backend.services.datastore.adapter:Get response with status code 200: {'ids': [35]}
backend_1           | [2021-06-14 08:56:58 +0000] [48] [ERROR] Error handling request /system/action/handle_request
backend_1           | Traceback (most recent call last):
backend_1           |   File "/usr/local/lib/python3.8/site-packages/gunicorn/workers/sync.py", line 136, in handle
backend_1           |     self.handle_request(listener, req, client, addr)
backend_1           |   File "/usr/local/lib/python3.8/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
backend_1           |     respiter = self.wsgi(environ, resp.start_response)
backend_1           |   File "/app/openslides_backend/http/application.py", line 121, in __call__
backend_1           |     return self.wsgi_application(environ, start_response)
backend_1           |   File "/app/openslides_backend/http/application.py", line 111, in wsgi_application
backend_1           |     response = self.dispatch_request(request)
backend_1           |   File "/app/openslides_backend/http/application.py", line 41, in dispatch_request
backend_1           |     return self.default_route(request)
backend_1           |   File "/app/openslides_backend/http/application.py", line 68, in default_route
backend_1           |     response_body, access_token = view_instance.dispatch(request)
backend_1           |   File "/app/openslides_backend/http/views.py", line 62, in dispatch
backend_1           |     response = handler.handle_request(request.json, user_id, is_atomic)
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 98, in handle_request
backend_1           |     results = self.execute_write_requests(self.parse_actions, payload)
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 131, in execute_write_requests
backend_1           |     write_requests, data = get_write_requests(*args)
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 168, in parse_actions
backend_1           |     write_request, results = self.perform_action(element, relation_manager)
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 202, in perform_action
backend_1           |     write_request, results = action.perform(action_data, self.user_id)
backend_1           |   File "/app/openslides_backend/action/action.py", line 145, in perform
backend_1           |     instance = self.base_update_instance(instance)
backend_1           |   File "/app/openslides_backend/action/generics/create.py", line 24, in base_update_instance
backend_1           |     instance = self.update_instance(instance)
backend_1           |   File "/app/openslides_backend/action/actions/mediafile/create_directory.py", line 46, in update_instance
backend_1           |     instance["inherited_access_group_ids"] = instance["access_group_ids"]
backend_1           | KeyError: 'access_group_ids'
```

Do you need more information?

--- a/openslides_backend/action/actions/mediafile/calculate_mixins.py
+++ b/openslides_backend/action/actions/mediafile/calculate_mixins.py
@@ -13,10 +13,10 @@ class MediafileCalculatedFieldsMixin(BaseAction):
 
     def calculate_inherited_groups(
         self,
-        access_group_ids: List[int],
+        access_group_ids: Optional[List[int]],
         parent_is_public: Optional[bool],
         parent_inherited_access_group_ids: Optional[List[int]],
-    ) -> Tuple[bool, List[int]]:
+    ) -> Tuple[bool, Optional[List[int]]]:
 
         parent: Dict[str, Any] = dict()
         parent["inherited_access_group_ids"] = parent_inherited_access_group_ids

--- a/openslides_backend/action/actions/mediafile/create_directory.py
+++ b/openslides_backend/action/actions/mediafile/create_directory.py
@@ -38,11 +38,11 @@ class MediafileUpdate(MediafileCalculatedFieldsMixin, CreateAction):
                 instance["is_public"],
                 instance["inherited_access_group_ids"],
             ) = self.calculate_inherited_groups(
-                instance["access_group_ids"],
+                instance.get("access_group_ids"),
                 parent.get("is_public"),
                 parent.get("inherited_access_group_ids"),
             )
         else:
-            instance["inherited_access_group_ids"] = instance["access_group_ids"]
+            instance["inherited_access_group_ids"] = instance.get("access_group_ids")
             instance["is_public"] = not bool(instance["inherited_access_group_ids"])
         return instance

--- a/tests/system/action/mediafile/test_create_directory.py
+++ b/tests/system/action/mediafile/test_create_directory.py
@@ -107,7 +107,6 @@ class MediafileCreateDirectoryActionTest(BaseActionTestCase):
             {
                 "mediafile/110": {
                     "title": "title_srtgb199",
-                    "child_ids": [111],
                     "access_group_ids": [],
                     "inherited_access_group_ids": [],
                     "is_public": True,
@@ -269,6 +268,28 @@ class MediafileCreateDirectoryActionTest(BaseActionTestCase):
         assert model.get("access_group_ids") == [3]
         assert model.get("inherited_access_group_ids") == []
         assert model.get("is_public") is False
+
+    def test_create_directory_no_access_group_ids(self) -> None:
+        self.set_models(
+            {
+                "meeting/110": {},
+            }
+        )
+        response = self.request(
+            "mediafile.create_directory",
+            {
+                "meeting_id": 110,
+                "title": "title_UoOFrRU3INADQbay",
+                "parent_id": None,
+            },
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("mediafile/1")
+        assert model.get("title") == "title_UoOFrRU3INADQbay"
+        assert model.get("access_group_ids") is None
+        assert model.get("inherited_access_group_ids") is None
+        assert model.get("is_directory") is True
+        assert model.get("is_public") is True
 
     def test_create_dictionary_no_permissions(self) -> None:
         self.base_permission_test(

